packages/components/table/src/table/style-helper.ts

```diff
@@ -153,22 +153,24 @@ function useStyle<T>(
   }, 10)
 
   const bindEvents = () => {
-    window.addEventListener('resize', doLayout)
     table.refs.bodyWrapper.addEventListener('scroll', syncPostion, {
       passive: true,
     })
     if (props.fit) {
       addResizeListener(table.vnode.el as ResizableElement, resizeListener)
+    } else {
+      window.addEventListener('resize', doLayout)
```

Comment:
Use `on` and `off` exported from `@element-plus/utils/dom`

Reply:
OK，I'm done 🙇